client/lib/domains/registration/availability-messages.js

```diff
@@ -20,6 +20,10 @@ function getAvailabilityNotice( domain, error ) {
 
 	const tld = getTld( domain );
 
+	if ( error.error ) {
```

Comment:
Good catch, though I think this should be fixed in the caller, more precisely here: https://github.com/Automattic/wp-calypso/blob/e3d5a722cf1e6d303d70f2f37f601028164f3a86/client/components/domains/register-domain-step/index.jsx#L459. `error.error` should just be passed directly to this function there.

Reply:
Thanks, I have fixed this 👍